toothpick-runtime/src/main/java/toothpick/Toothpick.java

```diff
@@ -39,12 +39,35 @@
   // its transformation
   private static final ConcurrentHashMap<Object, Scope> MAP_KEY_TO_SCOPE =
       new ConcurrentHashMap<>();
+  // ConcurrentHashSet doesn't exist so we're not using a Set ;)
+  private static final ConcurrentHashMap<Object, Scope> ROOT_SCOPES = new ConcurrentHashMap<>();
   static Injector injector = new InjectorImpl();
 
   protected Toothpick() {
     throw new RuntimeException("Constructor can't be invoked even via reflection.");
   }
 
+  /**
+   * This function will return a root scope as follows:<br>
```

Comment:
Remove 'this function as it is a method in oop and we avoid totological reference in doc. -> Opens the root scope.